packages/nextjs/cypress/e2e/contract_interaction.cy.ts

```diff
@@ -0,0 +1,719 @@
+const PAREX_CONTRACT_ABI = `[
```

Comment:
Lets move this variable in other file lol it blocks the whole main code.  Also maybe rename is `PAREX_CONTRACT_ABI_INPUT` ?

Reply:
I moved it to fixtures folder. 
In the tests we import it with cy and stringify it to then pass it to importABI

Reply:
Love it! I think thats what fixtures are for? So that you have some manual hardcoded data to test out?